app/controllers/api/v1/users_controller.rb

```diff
@@ -3,6 +3,26 @@
 module Api
   module V1
     class UsersController < ApplicationController
+      def create
+        @user = User.new(user_params)
+        if @user.save
+          render json: UserSerializer.new(@user).serializable_hash.to_json, status: :created
+        else
+          render_error
+        end
+      end
+
+      private
+
+      def user_params
+        params.require(:user).permit(:first_name, :last_name, :email, :password_digest, :photo,
```

Comment:
Check if this is correct because `password_digest` is the field to store the encrypted password, but not the name of the user param
[Ruby on Rails docs](https://api.rubyonrails.org/classes/ActiveModel/SecurePassword/ClassMethods.html)

Reply:
Done!